obal/data/playbooks/nightly/nightly.yaml

```diff
@@ -3,9 +3,32 @@
     - packages
   serial: 1
   gather_facts: no
-  roles:
-    - build_package
   vars:
-    releasers:
-      - "{{ nightly_releaser }}"
-    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
+    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args | default([]) }}"
+  tasks:
```

Comment:
If you change this to `pre_tasks` then you can avoid the `import_role` below and just have a `roles` section, e.g.

```
pre_tasks:
 - stufffff
roles:
 - build_package
```

Reply:
oh interesting! is that preferred over `import_role`?